pyannote/core/segment.py

```diff
@@ -350,7 +350,10 @@ def _repr_png_(self):
         :mod:`pyannote.core.notebook`
         """
         from .notebook import repr_segment
-        return repr_segment(self)
+        try:
+            return repr_segment(self)
+        except ImportError:
+            return None
```

Comment:
Same. Would be nice to warn users about the existence of pyannote.core[notebook].